android/src/main/java/com/wix/interactable/InteractableView.java

```diff
@@ -421,6 +421,10 @@ public void setHorizontalOnly(boolean horizontalOnly) {
 
     public void setDragEnabled(boolean dragEnabled) {
         this.dragEnabled = dragEnabled;
+        
+        if (this.dragBehavior != null && !dragEnabled) {
```

Comment:
I'm not sure if the check for `this.dragBehavior != null` is necessary but I saw it being checked in other parts of the core.